#! /bin/sh /usr/share/dpatch/dpatch-run
## 16_fix_typos_webalizer.1.dpatch by Jose Carlos Medeiros <debian@psabs.com.br>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix typo errors in webalizer.1 man file

@DPATCH@
Index: webalizer-2.01.10-vc/webalizer.1
===================================================================
--- webalizer-2.01.10-vc.orig/webalizer.1	2006-10-27 14:31:16.000000000 +0400
+++ webalizer-2.01.10-vc/webalizer.1	2006-10-27 14:31:35.000000000 +0400
@@ -60,7 +60,7 @@
 .TP 8
 .B o
 If an output directory was specified, the program does a \fBchdir(2)\fP to
-that directory in prepration for generating output.  If no output
+that directory in preparation for generating output.  If no output
 directory was given, the current directory is used.
 .TP 8
 .B o
@@ -91,7 +91,7 @@
 .TP 8
 .B o
 Main processing begins on the log file.  If the log spans multiple
-months, a seperate HTML document is created for each month.
+months, a separate HTML document is created for each month.
 .TP 8
 .B o
 After main processing, the main \fIindex.html\fP page is created, which
@@ -114,7 +114,7 @@
 internal data in a file named \fIwebalizer.current\fP.  This allows the
 program to 'start where it left off' so to speak, and allows the
 preservation of detail from one run to the next.  The data file is
-placed in the current output directory, and is a plain ascii text
+placed in the current output directory, and is a plain ASCII text
 file that can be viewed with any standard text editor.  It's location
 and name may be changed using the \fIIncrementalName\fP configuration
 keyword.
@@ -137,7 +137,7 @@
 processed, or process logs that contain a mix of processed/not yet
 processed records, and not produce duplication of statistics.  The
 only time this may break is if you have duplicate timestamps in two
-seperate log files... any records in the second log file that do have
+separate log files... any records in the second log file that do have
 the same timestamp as the last record in the previous log file processed,
 will be discarded as if they had already been processed.  There are
 lots of ways to prevent this however, for example, stopping the web
@@ -182,11 +182,11 @@
 \fBIncremental\fP.  Preserve internal data between runs.
 .TP 8
 .B \-q
-\fBQuiet\fP.  Supress informational messages.  Does not supress
+\fBQuiet\fP.  Suppress informational messages.  Does not suppress
 warnings or errors.
 .TP 8
 .B \-Q
-\fBReallyQuiet\fP.  Supress all messages including warnings and errors.
+\fBReallyQuiet\fP.  Suppress all messages including warnings and errors.
 .TP 8
 .B \-T
 \fBTimeMe\fP.  Force display of timing information at end of processing.
@@ -215,10 +215,10 @@
 Normally, out of sequence log records are simply ignored.
 .TP 8
 .B \-Y
-\fBCountryGraph\fP. Supress country graph.
+\fBCountryGraph\fP. Suppress country graph.
 .TP 8
 .B \-G
-\fBHourlyGraph\fP.  Supress hourly graph.
+\fBHourlyGraph\fP.  Suppress hourly graph.
 .TP 8
 .B \-x \fIname\fP
 \fBHTMLExtension\fP.  Defines HTML file extension to use.  If not
@@ -226,10 +226,10 @@
 period.
 .TP 8
 .B \-H
-\fBHourlyStats\fP.  Supress hourly statistics.
+\fBHourlyStats\fP.  Suppress hourly statistics.
 .TP 8
 .B \-L
-\fBGraphLegend\fP.  Supress color coded graph legends.
+\fBGraphLegend\fP.  Suppress color coded graph legends.
 .TP 8
 .B \-l \fInum\fP
 \fBGraphLines\fP.  Specify number of background lines. Default
@@ -273,7 +273,7 @@
 .TP 12
 .ti 8
 .B 0
-All informaiton (left unchanged).
+All information (left unchanged).
 .TP 8
 .B \-g \fInum\fP
 \fBGroupDomains\fP. Automatically group sites by domain.  The
@@ -286,7 +286,7 @@
 .TP 8
 .B \-N \fInum\fP
 \fBDNSChildren\fP.  Use \fInum\fP DNS children processes to perform DNS
-lookups, either creating or updateing the DNS cache file.  Specify zero
+lookups, either creating or updating the DNS cache file.  Specify zero
 (\fB0\fP) to disable cache file creation/updates.  If given, a DNS cache
 filename must be specified.
 .PP
@@ -333,7 +333,7 @@
 Configuration files are standard \fBascii(7)\fP text files that may be created
 or edited using any standard editor.  Blank lines and lines that begin
 with a pound sign ('#') are ignored.  Any other lines are considered to
-be configurgation lines, and have the form "Keyword Value", where the
+be configuration lines, and have the form "Keyword Value", where the
 \'Keyword\' is one of the currently available configuration keywords defined
 below, and 'Value' is the value to assign to that particular option.  Any
 text found after the keyword up to the end of the line is considered the
@@ -374,11 +374,11 @@
 in the '\fBTop URL's\fP' table.
 .TP 8
 .B Quiet \fP( yes | \fBno\fP )
-Supress informational messages.  Warning and Error messages will not be
+Suppress informational messages.  Warning and Error messages will not be
 supressed.
 .TP 8
 .B ReallyQuiet \fP( yes | \fBno\fP )
-Supress all messages, including Warning and Error messages.
+Suppress all messages, including Warning and Error messages.
 .TP 8
 .B Debug \fP( yes | \fBno\fP )
 Print extra debugging information on Warnings and Errors.
@@ -469,13 +469,13 @@
 Display the top \fInum\fP User Agents table. Use zero to disable.
 .TP 8
 .B AllAgents \fP( yes | \fBno\fP )
-Create seperate HTML page with \fBAll\fP User Agents.
+Create separate HTML page with \fBAll\fP User Agents.
 .TP 8
 .B TopReferrers \fInum\fP
 Display the top \fInum\fP Referrers table. Use zero to disable.
 .TP 8
 .B AllReferrers \fP( yes | \fBno\fP )
-Create seperate HTML page with \fBAll\fP Referrers.
+Create separate HTML page with \fBAll\fP Referrers.
 .TP 8
 .B TopSites \fInum\fP
 Display the top \fInum\fP Sites table. Use zero to disable.
@@ -484,7 +484,7 @@
 Display the top \fInum\fP Sites (by KByte) table.  Use zero to disable.
 .TP 8
 .B AllSites \fP( yes | \fBno\fP )
-Create seperate HTML page with \fBAll\fP Sites.
+Create separate HTML page with \fBAll\fP Sites.
 .TP 8
 .B TopURLs \fInum\fP
 Display the top \fInum\fP URLs table. Use zero to disable.
@@ -493,7 +493,7 @@
 Display the top \fInum\fP URLs (by KByte) table.  Use zero to disable.
 .TP 8
 .B AllURLs \fP( yes | \fBno\fP )
-Create seperate HTML page with \fBAll\fP URLs.
+Create separate HTML page with \fBAll\fP URLs.
 .TP 8
 .B TopCountries \fInum\fP
 Display the top \fInum\fP Countries in the table. Use zero to disable.
@@ -508,14 +508,14 @@
 Display the top \fInum\fP Search Strings in the table.  Use zero to disable.
 .TP 8
 .B AllSearchStr \fP( yes | \fBno\fP )
-Create seperate HTML page with \fBAll\fP Search Strings.
+Create separate HTML page with \fBAll\fP Search Strings.
 .TP 8
 .B TopUsers \fInum\fP
 Display the top \fInum\fP Usernames in the table.  Use zero to disable.
 Usernames are only available if using http based authentication.
 .TP 8
 .B AllUsers \fP( yes | \fBno\fP )
-Create seperate HTML page with \fBAll\fP Usernames.
+Create separate HTML page with \fBAll\fP Usernames.
 .PP
 .I Hide/Ignore/Group/Include Keywords
 .TP 8
@@ -651,7 +651,7 @@
 Dump the url data to a tab delimited file.
 .TP 8
 .B DumpReferrers \fP( yes | \fBno\fP )
-Dump the referrer data to a tab delimitd file.  This data is only
+Dump the referrer data to a tab delimit file.  This data is only
 available if using a log that contains referrer information
 (ie: a combined format web log).
 .TP 8
